custom-commands/hospitality/deployment/README.md

```diff
@@ -10,8 +10,25 @@
 * .NET Core SDK 2.1 or higher. [Install SDK here](https://docs.microsoft.com/en-us/dotnet/core/install/sdk?pivots=os-windows).
 
 ## What are you deploying?
- 
- TODO: Need diagram of CC and Visualization, and a description of the components and how they interact
+
+You will be deploying a sample [Custom command](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/custom-commands) project in your [Speech Studio](https://speech.microsoft.com/portal) portal. To demonstrate successful task completion (e.g. lights have turned off when you said "lights off"), we created a web page with a simulated scene. The completion rules in the Custom Command project are configured to make web calls to an Azure function that manages the state of the scene. Therefore in addition to the Custom Command project, you will be deploying all the Azure resources needed to support this visualization, in your Azure subscription.
+
+The following block diagram describes how the demo runs: 
+<p align="center">
+<img src="images/block-diagram.png"/>
+</a>
+</p>
+We assume that you read about Custom Commands and Speech SDK client applications, and here we describe the visualization part:
+
+* An Azure storage is created, which holds an HTML file and associated images needed to visualize all the states of the scene 
+* Azure storage also includes a table with one row, which is used to save the state of the scene 
+* You can view the HTML file in your web browser, since it will have a unique URL. It will show the default scene at first
+* An Azure function is created. It has access to the storage. When your Custom Command project has completed a dialog (e.g. "turn on the lights"), its completion rule will make a web call invoking the Azure function with query URL parameters indicating the desired new state of the scene (lights are on)
```

Comment:
This sounds like you are saying the state is in the query URL parameters which I believe is not true. The state gets passed in the body where the parameter is the room name

Reply:
Thanks, will fix